ChangeLog

```diff
@@ -1,3 +1,13 @@
+2015-04-01  Jacob Fenton  <bocajnotnef@gmail.com>
+   * scripts/test-{scripts.py, data/test-nonwritable}: added test for
```

Comment:
Not that it really matters: In glob syntax there is no space after the comma between options inside braces :-)